.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,17 @@
 {
-  "name": "Sample Development Environment for CCF",
-  "context": "..",
+  "name": "CCF Development Environment",
   "image": "ccfmsrc.azurecr.io/ccf/ci:oe-0.18.2-virtual",
   "runArgs": [],
-  "extensions": ["ms-vscode.cpptools", "ms-python.python"]
```

Comment:
It's probably worth keeping the python extension

Reply:
eek! I copied the wrong extension list! Thanks ever so much!!!